README.md

```diff
@@ -262,7 +262,11 @@ The following subcommands are supported:
       print           -- Print short description of the JavaScript value.
 
                          Syntax: v8 print expr
-      source          -- Source code information
+      source list     -- Print source lines around the currently selected 
```

Comment:
Do you remember why there are multiple words in this command? @indutny

Reply:
I imagine it's because lldb has a `source list` command with a similar purpose.